conf.yaml

```diff
@@ -901,6 +901,24 @@ contents:
               -
                 repo:   swiftype
                 path:   docs
+    -
+        title:      SIEM: Security Operations with the Elastic Stack
+        sections:
+          -
+            title:      SIEM Guide
+            prefix:     en/siem/guide
+            current:    7.2
+            branches:   [ master, 7.x, 7.2 ]
+            index:      docs/en/infraops/index.asciidoc
```

Comment:
Shouldn't this path include "siem"?

Reply:
Yup. Accidentally reverted while I was doing some testing. Thanks!